src/solc_0.7/common/BaseWithStorage/ERC721BaseToken.sol

```diff
@@ -259,26 +264,40 @@ contract ERC721BaseToken is IERC721, WithSuperOperators, WithMetaTransaction {
         return id == 0x01ffc9a7 || id == 0x80ac58cd;
     }
 
+    function _updateOwner(
```

Comment:
Maybe the namw should be `_updateOwnerData(` as in some cases (like `approve` the owner is not updated, only the operator flag.
Otherwise, this looks good (& it doesn't modify/overwrite the version number).

Reply:
I agree, I ll do that